pom.xml

```diff
@@ -189,10 +189,39 @@
         <artifactId>maven-surefire-plugin</artifactId>
         <version>2.19.1</version>
         <configuration>
-          <suiteXmlFiles>
-            <suiteXmlFile>src/test/java/ome/codecs/utests/testng.xml</suiteXmlFile>
-          </suiteXmlFiles>
+            <skip>true</skip>
         </configuration>
+        <executions>
+          <execution>
+            <id>default-tests</id>
+            <phase>test</phase>
+            <goals>
+              <goal>test</goal>
+            </goals>
+            <configuration>
+              <skip>false</skip>
+              <excludes>
+                <exclude>%regex[.*Missing.*Test.*]</exclude>
+              </excludes>
+            </configuration>
+          </execution>
+          <execution>
+            <id>missing-tests</id>
+            <phase>test</phase>
+            <goals>
+              <goal>test</goal>
+            </goals>
+            <configuration>
+              <skip>false</skip>
+              <includes>
+                <include>%regex[.*Missing.*Test.*]</include>
+              </includes>
+              <classpathDependencyExcludes>
+                 <classpathDependencyExcludes>org.openmicroscopy:ome-jai</classpathDependencyExcludes>
```

Comment:
You mean excluding our own stubs as well. That would makes sense for completeness, although the tests do not seem to require it interestingly.